portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -2063,4 +2544,60 @@ definition {
 			title = "WC Title");
 	}
 
+
+	@description = "LPS-122873 When the user add filters need to have only one vocabulary, the relative ordering of the categories remain the same"
+	@priority = "5"
```

Comment:
Also review the priority of all of the tests. All of them are Critical=FP5??
Remember:
* Critical: FP5
* High: FP4
* Medium: FP3